tests/WE2E/end_to_end_tests.sh

```diff
@@ -3,44 +3,55 @@
 #----------------------------------------------------------------------
 #  Automation of UFS Short Range Weather App Worfklow End to End Tests.
 #
-#  The script is dependant on a successful build of this repo. 
+#  The script is dependant on a successful build of this repo using the
+#  test/build.sh script in the ufs-srweather-app repository.
 #  The UFS build must be completed in a particular manner for this 
 #  script to function properly, notably the location of the build and
 #  bin directories: 
-#    BUILD_DIR=${APP_DIR}/build_${compiler}  
-#    BIN_DIR=${APP_DIR}/bin_${compiler}  
-#  
+#    BUILD_DIR=${APP_DIR}/build_${compiler}
+#    BIN_DIR=${APP_DIR}/bin_${compiler}
+#
 #  With the creating of a test_log directory and output files describing
 #  the result of each workflow status, a testing framework is established.
 #  The user must verify the contents of each experiment file in test_log/ 
-#  to check workflow success or failure.   
+#  to check workflow success or failure.
 #
-#  Example: . end_to_end_tests.sh hera zrtrr
+#  Example: ./end_to_end_tests.sh hera zrtrr
 #----------------------------------------------------------------------
 
 #-----------------------------------------------------------------------
 #  Set variables
 #-----------------------------------------------------------------------
 
-branches=( rrfs_ci ) 
+branches=( rrfs_ci )
+
+function usage {
+  echo
+  echo "Usage: $0 machine slurm_account  | -h"
+  echo
+  echo "       machine       [required] is one of: ${machines[@]}"
+  echo "       slurm_account [required] case sensitive name of the user-specific slurm account"
+  echo "       -h            display this help"
+  echo
+  exit 1
+
+}
+
+machines=( hera )
+
+if [ "$1" = "-h" ] ; then usage ; fi
+[[ $# -le 1 ]] && usage
 
 export machine=$1
-machine=`echo "${machine}" | tr '[A-Z]' '[a-z]'`  # scripts in sorc need lower case machine name
+machine=$(echo "${machine}" | tr '[A-Z]' '[a-z]')  # scripts in sorc need lower case machine name
 
 export account=$2
-account=`echo "${account}"`
-
 
 #-----------------------------------------------------------------------
-# Choose experiment. ( Full list in  ./baselines_list.txt)  
+# Choose experiment.
 #-----------------------------------------------------------------------
 
-#expts=( grid_RRFS_CONUS_25km_ics_HRRR_lbcs_RAP_suite_RRFS_v1alpha )
-expts=( grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2 )
-#expts=( grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2 )
-#expts=( grid_RRFS_CONUS_13km_ics_HRRR_lbcs_RAP_suite_HRRR )
-#expts=( inline_post )
-#expts=( grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2 grid_RRFS_CONUS_25km_ics_HRRR_lbcs_RAP_suite_RRFS_v1alpha )
+expts=( grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GSD_SAR )
```

Comment:
Just removing commented code here.

Reply:
I do agree with removing the commented code, but if we'd like to run more than one experiment in the future, I believe we should provide additional information or some sort of "skeleton". 

I left the extras in here to change which experiments to run. It is easy to toggle the commented experiments and re-run this script. As there were errors with finding IC/LBCS with particular experiments, I wanted to have a couple different experiments on deck.  I reckon it would be nice to have a way to submit multiple experiments easily.  

Do we only want to have one 'test' (experiment) for now?   

If we want to accommodate more than one experiment, do we want to provide those as a config file, as CLA, or uncommenting code?